main.rb

```diff
@@ -0,0 +1,4 @@
+require_relative 'lib/parser'
+
+file = Parser.new('games.log')
+file.get_first_line
```

Comment:
aqui dai vc chamaria
```suggestion
puts file.get_first_line
```